packages/engines/core-engine/src/game-engine/engines/lorcana/src/cards/definitions/008/actions/176-pouncing-practice.ts

```diff
@@ -26,14 +26,20 @@ export const pouncingPractice: LorcanaActionCardDefinition = {
   abilities: [
```

Comment:
Change this to a single ability with two effects. Instead of two abilities with one effect each